var/spack/repos/builtin/packages/findutils/package.py

```diff
@@ -51,6 +51,7 @@ def url_for_version(self, version):
     patch('nvhpc.patch', when='@4.6.0 %nvhpc')
     # Workaround bug where __LONG_WIDTH__ is not defined
     patch('nvhpc-long-width.patch', when='@4.8.0:4.8 %nvhpc')
+    patch('nonnull.patch')
```

Comment:
Thanks for the explanation, can you add a comment above this patch directive with a reference to https://savannah.gnu.org/bugs/?func=detailitem&item_id=59972 and a very brief description of the issue it solves?

Reply:
Done.